parallel-install/pkg/deployment/coredns_test.go

```diff
@@ -0,0 +1,64 @@
+package deployment
+
+import (
+	"testing"
+
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/logger"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func Test_patchCoreDNS(t *testing.T) {
+	log := logger.NewLogger(true)
+	coreDNSCM := fakeCoreDNSCM()
+
+	t.Run("test skipping coreDNS patch", func(t *testing.T) {
+		// given
+		kubeClient := fake.NewSimpleClientset(coreDNSCM)
+
+		// when
+		cm, err := patchCoreDNS(kubeClient, `(.*)\.loacal\.kyma\.dev`, log)
+
+		// then
+		require.NoError(t, err)
+		require.Empty(t, cm.Data)
+	})
+}
+
+func fakeCoreDNSCM() *v1.ConfigMap {
```

Comment:
I don't like the name while you combine two different words with uppercase - DNS and CM. Could you name it differently to make it more readable, like: `fakeCoreDnsCm` or `fakeCoreDNScm`?

Reply:
DNS has to be uppercase, renamed to `fakeCoreDNSConfigMap`